packages/lambda-api-tracker/src/__test__/validate.test.ts

```diff
@@ -0,0 +1,66 @@
+import { ApiKeyTable, LambdaSession, Logger } from '@basemaps/shared';
+import { ValidateRequest } from '../validate';
+
+describe('QueryString', (): void => {
+    const dummyApiKey = 'dummy1';
+    const faultyApiKey = 'fault1';
+    const mockApiKey = 'mock1';
+
+    beforeEach(() => {
+        LambdaSession.reset();
+    });
+
+    it('tests that apikeytable gets called', (): void => {
+        ApiKeyTable.prototype.get = jest.fn();
```

Comment:
This will destroy the `ApiKeyTable.get` for all future invocations of it, 

Using `jest.spyOn(AWS.api.db, 'get')` will mean that the `get` function is restored after every test.